python/lsst/ip/isr/isrTask.py

```diff
@@ -1728,6 +1752,36 @@ def extractCalibDate(calib):
         else:
             return "Unknown Unknown"
 
+    def compareCameraKeywords(self, exposureMetadata, calib, calibName):
+        """Compare header keywords to confirm camera states match.
+
+        Parameters
+        ----------
+        exposureMetadata : `lsst.daf.base.PropertySet`
+            Header for the exposure being processed.
+        calib : `lsst.afw.image.Exposure` or `lsst.ip.isr.IsrCalib`
+            Calibration to be applied.
+        calibName : `str`
+            Calib type for log message.
+        """
+        try:
+            calibMetadata = calib.getMetadata()
+        except AttributeError:
+            return
+        for keyword in self.config.cameraKeywordsToCompare:
+            if keyword in exposureMetadata and keyword in calibMetadata:
+                if exposureMetadata[keyword] != calibMetadata[keyword]:
+                    if self.config.doRaiseOnCalibMismatch:
+                        raise RuntimeError("Sequencer mismatch for %s [%s]: exposure: %s calib: %s",
+                                           calibName, keyword,
+                                           exposureMetadata[keyword], calibMetadata[keyword])
+                    else:
+                        self.log.warning("Sequencer mismatch for %s [%s]: exposure: %s calib: %s",
+                                         calibName, keyword,
+                                         exposureMetadata[keyword], calibMetadata[keyword])
+            else:
+                self.log.debug("Sequencer keyword %s not found.", keyword)
```

Comment:
This is debug as it will just be noise for other cameras.

Reply:
But my point was that here you're inside a `for keyword in self.config.cameraKeywordsToCompare` block, and that will be an empty list for all these cameras for which it would be noise. So given that to be here all, that config list is being set, doesn't that showing up constitute something quite bad happening?